src/components/QRShare/index.tsx

```diff
@@ -1,22 +1,31 @@
 import type {ForwardedRef} from 'react';
 import React, {forwardRef, useImperativeHandle, useRef, useState} from 'react';
 import type {LayoutChangeEvent} from 'react-native';
-import {View} from 'react-native';
+import {Platform, View} from 'react-native';
 import type {Svg} from 'react-native-svg';
 import ExpensifyWordmark from '@assets/images/expensify-wordmark.svg';
 import ImageSVG from '@components/ImageSVG';
 import QRCode from '@components/QRCode';
 import Text from '@components/Text';
+import useResponsiveLayout from '@hooks/useResponsiveLayout';
 import useTheme from '@hooks/useTheme';
 import useThemeStyles from '@hooks/useThemeStyles';
+import useWindowDimensions from '@hooks/useWindowDimensions';
+import * as Browser from '@libs/Browser';
 import variables from '@styles/variables';
 import type {QRShareHandle, QRShareProps} from './types';
 
 function QRShare({url, title, subtitle, logo, logoRatio, logoMarginRatio}: QRShareProps, ref: ForwardedRef<QRShareHandle>) {
     const styles = useThemeStyles();
     const theme = useTheme();
+    const {isSmallScreenWidth} = useResponsiveLayout();
+    const isMobilePlatform = Platform.OS === 'android' || Platform.OS === 'ios';
```

Comment:
Are 100% sure that we need Platform check here, isn't `isSmallScreenWidth` sufficient

Reply:
Thank you for your feedback. I checked and using `isSmallScreenWidth` is enough to detect the container width. I have updated the condition to get the container width. Please help me check it again!